staging/src/slime.io/slime/modules/lazyload/module/module.go

```diff
@@ -47,30 +66,17 @@ func (mo *Module) InitScheme(scheme *runtime.Scheme) error {
 	return nil
 }
 
-func (mo *Module) Clone() module.Module {
-	ret := *mo
+func (m *Module) Clone() module.Module {
+	ret := *m
 	return &ret
 }
 
 func (m *Module) Setup(opts module.ModuleOptions) error {
 
 	env, mgr := opts.Env, opts.Manager
-
-	pc, err := controllers.NewProducerConfig(env)
-	if err != nil {
-		return fmt.Errorf("unable to create ProducerConfig, %+v", err)
-	}
-	sfReconciler := controllers.NewReconciler(
-		controllers.ReconcilerWithCfg(&m.config),
-		controllers.ReconcilerWithEnv(env),
-		controllers.ReconcilerWithProducerConfig(pc),
-	)
-	sfReconciler.Client = mgr.GetClient()
-	sfReconciler.Scheme = mgr.GetScheme()
-
 	opts.InitCbs.AddStartup(func(ctx context.Context) {
```

Comment:
why we still add callbacks in `setup`/`run`?

Reply:
same with above